containers/daap-push-to-catalogue/src/var/task/push_to_catalogue.py

```diff
@@ -0,0 +1,70 @@
+import json
+import logging
+import os
+
+import boto3
+from data_platform_catalogue import CatalogueClient, CatalogueError
+from data_platform_catalogue import DataProductMetadata as omdProductMetadata
+from data_platform_catalogue import TableMetadata
+
+
+def handler(event, context):
+    # can't use daap-python-base as it's python 3.11 and need 3.10 for
+    # data_platform_catalogue, hence we can't use DataPlatformLogger here
+    logger = logging.getLogger()
+    metadata = event["metadata"]
+    version = event["version"]
+    data_product_name = event["data_product_name"]
+    table_name = event["table_name"]
+
+    logger.info(event)
+
+    secrets_client = boto3.client("secretsmanager")
+    jwt_secret = json.loads(
+        secrets_client.get_secret_value(
+            SecretId=os.getenv("OPENMETADATA_JWT_SECRET_ARN")
+        )["SecretString"]
+    )
+    token = jwt_secret["token"]
+    openmetadata_client = CatalogueClient(
+        jwt_token=token,
+        api_uri=os.getenv("OPENMETADATA_DEV_API_URL"),
```

Comment:
```suggestion
        api_uri=os.getenv("OPENMETADATA_API_URL"),
```
Are we expecting to set / pass this in as an env in the Dockerfile? Presumably this will only be `DEV` for now, and in future will be one of [dev, prod, preprod, test]?

Reply:
Nah, i'm setting it in the terraform and making available in the lambda function env vars. This doesn't refer to the accounts. I wasn't sure if we'd retain a dev instance of openmetadata or not hence i thought it should have a explicit dev name for now